src/libsyntax_pos/symbol.rs

```diff
@@ -515,7 +515,7 @@ impl LocalInternedString {
         }
     }
 
-    pub fn get(&self) -> &'static str {
+    pub fn get(&self) -> &str {
```

Comment:
Please add a comment and maybe an `ui-fulldeps` test, to ensure this doesn't happen again?
And maybe `self.string` should be `*const str` instead.

Reply:
I made this use unsafe code. That should prevent people from modifying this without reading comments.